index.html

```diff
@@ -123,13 +124,66 @@ <h5><a target="_blank" href="https://github.com/pcengines/apu2-documentation/blo
 
 
         <div class="container">
-            <!-- mainline releases v4.8.0.2 -->
+            <!-- mainline releases v4.8.0.4 -->
  
 <!--<div class="break"></div>insert this code when a new update comes out to separate a new version from the current.-->
 
-<div class="smaller-margin" id="mr-13">
+        <div class="smaller-margin" id="mr-13">
```

Comment:
v4.8.0.4 should have new id. @mscibisz `mr-13` is  v4.8.0.3 id. And I do not see v4.8.0.4 on the list on top of the site